src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -2626,6 +2629,17 @@ namespace HeatBalFiniteDiffManager {
         updatedThermalConductivity = materialDefinition.phaseChange->getConductivity(temperatureUpdated);
     }
 
+    bool findAnySurfacesUsingConstructionAndCondFD(EnergyPlusData &state, int const constructionNum)
+    {
+        for (int surfNum = 1; surfNum <= state.dataSurface->TotSurfaces; ++surfNum) {
+            auto &thisSurface = state.dataSurface->Surface(surfNum);
+            if (thisSurface.Construction == constructionNum) {
+                if (thisSurface.HeatTransferAlgorithm == DataSurfaces::HeatTransferModel::CondFD) return true;
+            }
+        }
```

Comment:
for (auto thisSurface : state.dataSurface->Surface) {
            if (thisSurface.Construction == constructionNum) {
                if (thisSurface.HeatTransferAlgorithm == DataSurfaces::HeatTransferModel::CondFD) return true;
            }
        }

Reply:
Ok--made this change as suggested and pushed the commit.